pythonarcade/arcade
# Add Contributing Guide index page

### Changes

* Add a contributing guide index page with a stubbed toctree to include future children
* Link the new page from the quick index
* Change the home page's Source Code heading to Source Code & Contributing

### Why

This is a first step to the following:
1. Centralizing contributor / developer install instructions 
2. Adding more detailed documentation covering our style conventions, unit tests, build system, etc
3. Adding short guides & links to general how-to pages for common development tasks & tools (ie adding remotes to check out & test other people's PRs)

### How to Test

1. Check out the branch locally
2. `./make.py serve`
3. Open http://localhost:8000 in your browser
3. Make sure the index page has been linked on the home page
4. Click the Contributing Guide index link to make sure it works
5. Make sure the content of the page looks ok
6. Click the CONTRIBUTING.md link to make sure it redirects users to GitHub for the quickstart.
